os/drivers/audio/syu645bscripts.h

```diff
@@ -45,7 +45,7 @@ typedef struct {
 
 t_codec_init_script_entry codec_reset_script[] = {
 	/* 0x0F for soft reset */
-	{ 0x0F, {0x01,}, 0, SYU645B_REG_D_2BYTE}
+	{ 0x0F, {0x01,}, 30, SYU645B_REG_D_2BYTE}
```

Comment:
I was uanble to find where they mentioned the delay after reset as 30ms. It looks like we can read the register until the bit is set to 0 once reset is done. Please check it later . It might be much faster than 30ms.